### DISCUSSION TITLE
fluidd and your fork

### DISCUSSION THREAD

**Discussion by bdoriano** [Desuuuu/klipper#90] [Q&A]
i have no idea of the way to install your fork on fluidd can you help me ?

**Comment by Desuuuu** [comment]
I'm pretty sure some people are using this fork with Fluidd but I'm not exactly familiar with it.

Essentially, you need to clone Klipper from this repo instead of the official one. I don't know how that translates to Fluidd though.

**Comment by HcNguyen111** [comment]
After you get Fluidd all set up (I assume FluiddOS - which comes with everything already "pre-installed") you'll need to SSH in and run this to remove the existing Klipper:

    rm -rf ~/klipper

then run these to replace with Desuuuu's fork:

    cd ~
    git clone https://github.com/Desuuuu/klipper.git

or, if you plan on using [the new version of Desuuuu's fork](https://github.com/Desuuuu/klipper/discussions/74):

    cd ~
    git clone -b dgus-display https://github.com/Desuuuu/klipper.git

Make sure to also edit the `[update_manager]` section in `moonraker.conf` so that Fluidd "accepts" this unofficial fork and can update it, like so:

    [update_manager]
    channel: dev
    enable_repo_debug: True
    enable_auto_refresh: True
    enable_system_updates: True

*Reminder to Restart Fluidd/the Pi after all these changes.*
